lib/core/text/flat.nit

```diff
@@ -582,6 +580,9 @@ class FlatBuffer
 
 	private var capacity = 0
 
+	# Real items, used as cache for when to_cstring is called
+	private var real_items: nullable NativeString = null
```

Comment:
I can do that yup, though I suspect it will make little to no difference since `to_cstring` is very rarely called on a `FlatBuffer`.

Still it's not a bad idea to optimize it, just in case.